src/deepsparse/transformers/pipelines/text_generation.py

```diff
@@ -63,7 +64,7 @@
 
 class GenerationDefaults:
```

Comment:
I'd like if we could restructure and comment this class such as 
```python
# Based off of https://huggingface.co/docs/transformers/main_classes/text_generation#transformers.GenerationConfig # noqa E501
class GenerationDefaults:
    # Parameters that control the length of the output
    max_length = None
    max_new_tokens = 100
    # Parameters that control the generation strategy used
    do_sample = False
    # Parameters for manipulation of the model output logits
    temperature = 1.0
    top_k = 50
    top_p = 1.0
    repetition_penalty = 1.0
    # Parameters that define the outputs
    num_return_sequences = 1
    output_scores = False
```

Reply:
updating `max_length` and `max_new_tokens` within the scope of this PR